src/app/component/layout/components/search-popup/search-not-found/search-not-found.component.ts

```diff
@@ -7,6 +7,7 @@ import {Component, EventEmitter, OnInit, Output} from '@angular/core';
 })
 export class SearchNotFoundComponent implements OnInit {
   @Output() closeUnsuccessfulSearchResults  = new EventEmitter();
+  @Input() inputValue;
 
   private emitClose(): void {
     this.closeUnsuccessfulSearchResults.emit(true);
```

Comment:
1. It shouldn't be `private`, because you use it in template. So, the build will be broken in prod mode with AOT.
2. Order method description appropriately - all methods below `constructor` and `ngOnInit`, or better yet - remove them if they are empty.

Reply:
fixed